spec/features/idv/doc_auth/how_to_verify_spec.rb

```diff
@@ -119,8 +119,10 @@
    when the sp has opted into ipp' do
     context 'opt in false at start but true during navigation' do
       it 'should be bounced back from Hybrid Handoff to How to Verify' do
+        sleep(5)
         expect(page).to have_current_path(idv_hybrid_handoff_url)
         allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { true }
+        sleep(5)
```

Comment:
Are these sleeps necessary for the test to pass, or were they debugging? Some of the helpers accept a `wait: 5` parameter to wait _up to_ 5 seconds for the content to appear without introducing a fixed sleep.

Reply:
@n1zyy , no still working on tests.

Reply:
@n1zyy  updated tests.